site/content/docs/5.3/customize/css-variables.md

```diff
@@ -57,14 +57,6 @@ Have a look at our table documentation for some [insight into how we're using CS
 
 Whenever possible, we'll assign CSS variables at the base component level (e.g., `.navbar` for navbar and its sub-components). This reduces guessing on where and how to customize, and allows for easy modifications by our team in future updates.
 
-### Dark text rule
-
-Using CSS variables allows a better flexibility on certain rules. This way we adapt our text variant color to our dark variants components instead of rewriting rules for each component. This process implies to override some existing CSS variables and to re-initialize those in some cases.
```

Comment:
Should we deprecate this instead of removing it ? Otherwise, we have to check broken links.